src/celeritas/ext/detail/PhysicsListUtils.hh

```diff
@@ -0,0 +1,17 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/ext/detail/PhysicsListUtils.hh
```

Comment:
What's this guy for?

Reply:
Oops, forgot to finish one additional minor change 😳 will do that before merging.

Reply:
Looks great!